### DISCUSSION TITLE
I wrapped your code in a Tkinter GUI

### DISCUSSION THREAD

**Discussion by sm18lr88** [DarkTrick/python-video-silence-cutter#13] [Show and tell]
I'm not sure how to properly attribute work. I really found your code useful, so I made a [GUI](https://github.com/sm18lr88/video-silence-cutter-GUI) for it.
Check it out if you'd like. Let me know if there's any issues or comments.

**Comment by DarkTrick** [comment]
Uh, that looks good! There's no source code in the repo. Is that on purpose? 

I could very well imagine merging the GUI into this repo here.

**Reply by sm18lr88** [reply]
Woops, forgot to drag that one file.
I don't know how to do pulls and requests in Github. I'm a total noob.

Check it out and if you like it, feel free to just put the file in your repo. 

Disclaimer: all I did was take your code and ask GPT-4 to create this tkinter app, and refined it several times.
